src/TeamSpeak3QueryApi/FileTransfer/FileTransferClient.cs

```diff
@@ -20,43 +20,43 @@ public int GetFileTransferId()
             return _currentFileTransferId++;
         }
 
-        public async Task SendFile(byte[] data, int port, string key)
+        public async Task SendFileAsync(byte[] data, int port, string key)
         {
             using var client = new TcpClient();
             await client.ConnectAsync(_host, port).ConfigureAwait(false);
             using var ns = client.GetStream();
 
-            // Send key
+            // SendAsync key
             var keyBytes = Encoding.ASCII.GetBytes(key);
             await ns.WriteAsync(keyBytes, 0, keyBytes.Length).ConfigureAwait(false);
             await ns.FlushAsync().ConfigureAwait(false);
 
-            // Send data
+            // SendAsync data
             await ns.WriteAsync(data, 0, data.Length).ConfigureAwait(false);
         }
 
-        public async Task SendFile(Stream dataStream, int port, string key)
+        public async Task SendFileAsync(Stream dataStream, int port, string key)
         {
             using var client = new TcpClient();
             await client.ConnectAsync(_host, port).ConfigureAwait(false);
             using var ns = client.GetStream();
 
-            // Send key
+            // SendAsync key
             var keyBytes = Encoding.ASCII.GetBytes(key);
             await ns.WriteAsync(keyBytes, 0, keyBytes.Length).ConfigureAwait(false);
             await ns.FlushAsync().ConfigureAwait(false);
 
-            // Send data
+            // SendAsync data
```

Comment:
Acceidental renames?

Reply:
I think yes. Every async method should be changed. When we have breaking changes then we shouldn't make some quick and dirty solutions